### DISCUSSION TITLE
Server Names

### DISCUSSION THREAD

**Discussion by lloricode** [spatie/ray#581] [Ideas]
Hi, 

It is better to have name or label on Servers, In my side I have many connection, and but confusing which is the right one to connect

Thank you

**Comment by AdrianMrn** [comment]
We will add this in the next version!

**Comment by AdrianMrn** [comment]
👀 
![image](https://user-images.githubusercontent.com/12762044/139456293-8f236ad6-465b-4250-ac07-ddcec1e11b45.png)

**Reply by lloricode** [reply]
Great! Thanks for the response <3

**Reply by lloricode** [reply]
hi @AdrianMrn Thank for the update, then by the way, after getting an update i was not able to connect
![Screenshot from 2021-11-05 14-27-04](https://user-images.githubusercontent.com/8251344/140467953-917b54bc-095b-474d-9c2d-eb02067ed00f.png)

**Reply by AdrianMrn** [reply]
My bad, releasing a fix now, 1.18.2 should be live in about 15 minutes :)

**Reply by lloricode** [reply]
> My bad, releasing a fix now, 1.18.2 should be live in about 15 minutes :)

thank you
